images/README.md

```diff
@@ -0,0 +1,110 @@
+# Building Images
+
+This directory contains tooling for building base images for use as nodes in Kubernetes Clusters. [Packer](https://www.packer.io) is used for building these images.
+
+## Prerequisites
+
+### Prerequisites for all images
+
+- [Packer](https://www.packer.io/docs/installation.html)
+- [Ansible](http://docs.ansible.com/ansible/latest/intro_installation.html) version >= 2.4.0.0
+
+### Prerequisites for Amazon Web Services
+
+- An AWS account
+- The AWS CLI installed and configured
+
+## Building Images
+
+### Build Variables
+
+The following variables can be overriden when building images using the `-var` option when calling `packer build`:
+
+| Variable | Default | Description |
+|----------|---------|-------------|
+| build_version | unset | A unique build version for the image |
```

Comment:
Lets go ahead and kill build_version in favor of using a timestamp instead. That way we can avoid requiring a user to manually specify it for each image built.

We can always re-add it in the future if using a timestamp becomes an issue.

Reply:
Sounds good